### DISCUSSION TITLE
Hide SSH banner when using docker image?

### DISCUSSION THREAD

**Discussion by cpainchaud** [drakkan/sftpgo#1215] [Q&A] [answered]
Hi community!

I am looking for the right flag to hide SSH banner which shows vendor + version. It's a valuable piece of info for a hacker....

```
[root@labdocker ~]# nc sftpgo.lab.local 2022
SSH-2.0-SFTPGo_2.4.4
```

thank you!!

**Comment by drakkan** [comment]
Hello,

you can change the banner, check the docs [here](https://github.com/drakkan/sftpgo/blob/main/docs/full-configuration.md)

**Comment by cpainchaud** [comment, marked as answer]
I failed on a typo, I should have tried harder :)

For the record this is what I used:

`-e SFTPGO_SFTPD__BANNER=WHATEVERYOUWANTHERE`
